### ISSUE TITLE
libxl-mac-latest.tar.gz 404 not found

### ISSUE THREAD

**Issue by gajus** [DirtyHairy/node-libxl#28]
Installation is failing with error:

```
Downloading https://www.libxl.com/download/libxl-mac-latest.tar.gz
unable to download the libxl SDK: request failed: 404 Not Found

```

it looks like download path has changed.

**Comment by DirtyHairy** [comment]
Hi gajus!

I just checked; worse: seems they switched the format to dmg. I'll do an update asap, but it might take a few days. In the meantime, you can use an old archive or tar up the SDK dir in the dmg and supply that to the install script via the `NODE_LIBXL_SDK_ARCHIVE` environment variable.

Cheers
    -Christian

**Comment by DirtyHairy** [comment]
Released new versions for 0.2, 0.3 and 0.4 with fixes.

**Comment by gajus** [comment]
Thank you
